docs/csharp/language-reference/compiler-messages/cs8177.md

```diff
@@ -49,7 +49,7 @@ class C
 
 ## To correct this error
 
-Changing the variable declaration to remove the `ref` modifier corrects this error:
+Changing the variable declaration to remove the `ref` modifier corrects this error. Or, you can upgrade to C# 13, which ships with .NET 9.
```

Comment:
Should this be "change"
```suggestion
Change the variable declaration to remove the `ref` modifier corrects this error. Or, you can upgrade to C# 13, which ships with .NET 9.
```
Or better yet, what about just directly stating the action since the heading already says this corrects the error:
```suggestion
Remove the `ref` modifier. Or, you can upgrade to C# 13, which ships with .NET 9.
```